client/blocks/comment-detail/style.scss

```diff
@@ -20,7 +20,8 @@
 	// If the comment is unapproved and collapsed, color it yellow.
 	&.is-unapproved.is-collapsed {
 		.comment-detail__header {
-			background: rgba( $alert-yellow, 0.085 );
+			//background: rgba( $alert-yellow, 0.085 );
+			background: mix( $alert-yellow, $white, 8.5% );
```

Comment:
I had to look this one up :) http://sass-lang.com/documentation/Sass/Script/Functions.html#mix-instance_method